packages/trie/README.md

```diff
@@ -137,11 +160,18 @@ By default, the deletion of trie nodes from the underlying database does not occ
 You can enable persistence by setting the `useRootPersistence` option to `true` when constructing a trie through the `Trie.create` function. As such, this value is preserved when creating copies of the trie and is incapable of being modified once a trie is instantiated.
 
 ```ts
+// ./examples/rootPersistence.ts
+
 import { Trie } from '@ethereumjs/trie'
+import { bytesToHex } from '@ethereumjs/util'
 
-const trie = await Trie.create({
-  useRootPersistence: true,
-})
+async function main() {
+  const trie = await Trie.create({
+    useRootPersistence: true,
+  })
+  console.log(bytesToHex(trie.root())) // 0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421
+}
```

Comment:
This example does not explain what this actually does (it saves the trie root, which in this example is the empty trie root, but it could also be some "weird other value")

Reply:
I added some more details on what's happening in the example.